catalog/transformer/catalog-transformer-zip/src/main/java/org/codice/ddf/catalog/transformer/zip/ZipValidator.java

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.transformer.zip;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.security.cert.Certificate;
+import java.security.cert.X509Certificate;
+import java.util.Enumeration;
+import java.util.Vector;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+import java.util.jar.Manifest;
+
+import org.apache.wss4j.common.crypto.Merlin;
+import org.apache.wss4j.common.ext.WSSecurityException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.security.PropertiesLoader;
+
+public class ZipValidator {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ZipValidator.class);
+
+    private String signaturePropertiesPath;
+
+    private static Merlin merlin;
```

Comment:
Fixed; this shouldn't be used statically.  Some other merlin uses in DDF: 
https://github.com/codice/ddf/blob/e5f90b2999b15e1bd1867cdeabc43554c0c483eb/platform/security/handler/security-handler-api/src/main/java/org/codice/ddf/security/handler/api/PKIAuthenticationTokenFactory.java

https://github.com/codice/ddf/blob/e5f90b2999b15e1bd1867cdeabc43554c0c483eb/platform/security/sts/security-sts-x509validator/src/main/java/org/codice/ddf/security/validator/x509/X509PathTokenValidator.java